src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -108,6 +118,9 @@ public final class Opt {
           + "Jazzer will create a temporary file and pass it to subprocesses.");
   public static final OptItem<List<String>> ignore = stringListSetting("ignore", ',',
       "Hex strings representing deduplication tokens of findings that should be ignored");
+  public static final OptItem<List<String>> instrument = OptParser.stringListSetting("instrument",
+      ',',
+      "Glob patterns matching names of classes that should be instrumented for fuzzing. Only used for JUnit fuzz tests");
```

Comment:
Mention that this combines `instrumentation_includes` and `custom_hook_includes`?

Reply:
Done!